<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-08C644NB" failures="1" tests="5" name="Create Customer API Testng" time="1.634" errors="0" timestamp="2022-08-27T11:04:16 IST">
  <testcase classname="com.w2a.APTestingFramework.testcases.CreateCustomerTest" name="invalidCreateCustomerAPI" time="1.275">
    <failure type="java.lang.AssertionError" message="expected [200] but found [401]">
      <![CDATA[java.lang.AssertionError: expected [200] but found [401]
at com.w2a.APTestingFramework.testcases.CreateCustomerAPI.sendPostrequesttoCreateCustomerAPIwithInvalidAuth(CreateCustomerAPI.java:43)
at com.w2a.APTestingFramework.testcases.CreateCustomerTest.invalidCreateCustomerAPI(CreateCustomerTest.java:39)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:539)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- invalidCreateCustomerAPI -->
  <testcase classname="com.w2a.APTestingFramework.testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="1.533"/>
  <testcase classname="com.w2a.APTestingFramework.testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="1.533"/>
  <testcase classname="com.w2a.APTestingFramework.testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="1.533"/>
  <testcase classname="com.w2a.APTestingFramework.testcases.CreateCustomerTest" name="validateCreateCustomerAPI" time="1.583"/>
</testsuite> <!-- Create Customer API Testng -->
